docs/source/getting_started/quick_start.rst

```diff
@@ -96,14 +100,11 @@ e. Set Up Individual Modules
 
         * Set the `Input Molecule File`: select the molecule you created above.
         * Adjust other settings as shown below. 
-        * Only for test runs:
```

Comment:
We have a similar remark in the ESAnalysis tool. Why delete here?

Reply:
Adjust other settings already covers this. below is the image where those flags are enables on the productio runs and disabled for test runs. So this is just a duplication.

Reply:
Then let's kill the "Adjust other settings" line, as both boxes are checked anyway when you load the WaNo, and keep the commend about test runs

Reply:
okay